tsconfig.json

```diff
@@ -1,12 +1,11 @@
 {
   "compilerOptions": {
-    "target": "ES2019" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "target": "es2022" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
```

Comment:
Nit: I suspect doesn't matter, was thinking it could be uppercase `ES2022`

Reply:
Yeah it's case insensitive